core/src/main/java/org/apache/spark/h2o/JavaH2OContext.java

```diff
@@ -45,16 +48,16 @@
  * can be set in H2O configuration class H2OConf
  *
  */
-public class JavaH2OContext {
+public class JavaH2OContext implements Serializable {
```

Comment:
This is dangerous! Why JavaH2OContext/H2OContext needs to be `Serializable`? I was looking at `SparkContext` it is not serializable (it is by design since you do not want to pass it around cluster).